pkg/deviceshifu/deviceshifumqtt/deviceshifumqtt_test.go

```diff
@@ -36,6 +57,28 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		klog.Fatal(err)
 	}
+
+	stop <- struct{}{}
+	wg.Wait()
+	os.Unsetenv("SERVER_LISTEN_PORT")
```

Comment:
can we use similar approach as PR https://github.com/Edgenesis/shifu/pull/375

Reply:
Nice!  I have already used it